builtin/logical/pki/path_acme_eab.go

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package pki
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+/*
+ * This file unlike the other path_acme_xxx.go are VAULT APIs to manage the
+ * ACME External Account Bindings, this isn't providing any APIs that an ACME
+ * client would use.
+ */
+func pathAcmeEabCreateList(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "acme/eab",
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixPKI,
+		},
+
+		Fields: map[string]*framework.FieldSchema{},
+
+		Operations: map[logical.Operation]framework.OperationHandler{
```

Comment:
Should these be forwarded by default? I think perhaps the one exception could be `list`, but `create+delete` will definitely need to be forwarded.

Reply:
Fixed in https://github.com/hashicorp/vault/commit/21300b4e9037174427fd69104c1aa3bbac37baaf